src/Sylius/Bundle/PayumBundle/Resources/views/layout.html.twig

```diff
@@ -1,4 +1,4 @@
-{% extends 'SyliusWebBundle:Frontend:layout.html.twig' %}
+{% extends 'SyliusShopBundle::layout.html.twig' %}
```

Comment:
:+1: - nevertheless I'm just moving the PayumBundle coupling from WebBundle to ShopBundle, it should be fixed in another PR.